content/docs/rfcs/24/README.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: 24
+title: 24/WAKU-VOTING
+name: Waku v2 Voting
+status: raw
+editor: Szymon Szlachtowicz <szymon.s@ethworks.io>
+---
+
+This spec is a proposition for a voting protocol over Waku V2.
+
+# Motivation
+
+In open p2p protocol there is an issue with voting off-chain as there is much room for malicious peers to only include votes that support their case when submitting votes to chain.
+
+Proposed solution is to agregate votes over waku and allow users to submit votes to smart contract that aren't already submitted.
+
+# Smart contract
+
+Voting should be finalized on chain so that the finished vote is immutable. Becouse of that, smart contract needs to be deployed. New smart contract will have to be able to start a vote and accept submited votes.
+
+## Double voting
+
+Smart contract should also keep a list of all votes so that no one can double vote.
+
+# Voting
+
+## Sending votes
+
+Sending votes is simple every peer is able to send a message to Waku topic specific to given application: 
+```
+/{application-name}/{voting-room}
+```
+
+vote object that is sent over waku should contain information about: 
+
+```ts
+{
```

Comment:
There should also be a vote information object if someone want to create a vote that contains the details of what is being voted on.

Reply:
Do you mean for the curation dapp or for general purpose? For now, let's focus the spec on curation. We can make a more generic one at a later stage.

See https://github.com/Szymx95/rfc/pull/1